app/controllers/application_controller.rb

```diff
@@ -1,3 +1,30 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  before_action :load_popular_users
```

Comment:
Don't need this for this app ... again: When copy-and-pasting things, make sure you know what it does and if you need it.